prow/gerrit/adapter/adapter.go

```diff
@@ -208,8 +241,7 @@ func (c *Controller) syncChange(latest client.LastSyncState, changeChan <-chan C
 			"revision": change.CurrentRevision,
 		})
 
-		log.WithField("duration(s)", time.Since(tracker).Seconds()).Debug("Time taken for gerrit change to be picked up by a worker thread.")
```

Comment:
Why are we removing this? This seems like a useful metric for us to instrument.

Reply:
For cleaning up log-based metrics (but I agree with your last comment that we may need to add more metrics).

I can add this one back (as a metric) in this PR if you prefer, or in a follow-up PR. Which would you prefer?

Reply:
Lets add that one back now if you don't mind. That one would be useful to help us tweak the worker thread count which I'm trying to start tonight.